mandatory/2-function-creation.js

```diff
@@ -16,7 +19,12 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) {
+    return "100%";
```

Comment:
Could you make one return instead of two returns , I think it will be prettier :-)